docs/src/AvatarGroup.doc.js

```diff
@@ -0,0 +1,551 @@
+        description: `Callback fired when the component is clicked (pressed and released) with a mouse or keyboard. See the [role](#Role) variant to learn more.`,
+      },
+      {
+        name: 'href',
+        type: 'string',
+        description: `When supplied, wraps the component in a link, and directs users to the url when item is selected. See the [role](#Role) variant to learn more.`,
+      },
+      {
+        name: 'addCollaborators',
+        type: 'boolean',
+        description: `When supplied, it appends an 'add' icon to the avatar pile as a call to action to the user. See [Best Practices] (#Best-practices) for more info.`,
+      },
+      {
+        name: 'ref',
+        type: `React.Ref<'div'> | React.Ref<'a'>`,
+        description: `Forward the ref to the underlying div or anchor element. See the [role](#Role) variant to learn more.`,
+        href: 'ref',
+      },
+    ]}
+  />,
+);
+
+card(
+  <MainSection name="Accessibility">
+    <MainSection.Subsection
+      title="ARIA attributes"
+      description={`
+AvatarGroup requires \`accessibilityLabel\`. AvatarGroup is a group of elements that require a parent label describing both the data presented and the call to action in the case of button and link roles. As seen in the example below, the screen-reader reads: "Group collaborators: Keerthi, Alberto, and another 10 collaborators. Add collaborators to this board."
+
+
+If AvatarGroup is used as a control button to show/hide Popover-component, we recommend passing the following ARIA attributes to assist screen readers:
+
+- \`accessibilityControls\`: informs the screen reader that AvatarGroup controls the display of an anchored Popover-component. It populates [aria-controls](https://www.w3.org/TR/wai-aria-practices/examples/menu-button/menu-button-links.html).
+- \`accessibilityHaspopup\`: informs the screen reader that there’s a Popover-component attached to AvatarGroup. It populates [aria-haspopup](https://www.w3.org/TR/wai-aria-practices/examples/menu-button/menu-button-links.html).
+- \`accessibilityExpanded\`: informs the screen reader whether an anchored Popover-component is currently open or closed. It populates [aria-expanded](https://www.w3.org/TR/wai-aria-practices/examples/menu-button/menu-button-links.html).
+`}
+    >
+      <MainSection.Card
+        cardSize="lg"
+        defaultCode={`
+function Example() {
+  const [open, setOpen] = React.useState(false);
+  const [selectedBoard, setSelectedBoard] = React.useState('Fashion');
+  const anchorRef = React.useRef();
+
+  const SearchCollaboratorsField = () => {
+    const ref = React.useRef();
+
+    React.useEffect(() => {
+      ref.current.focus();
+    }, []);
+
+    return (
+      <SearchField
+        accessibilityLabel="Search other users by name or email"
```

Comment:
This gets redundant, so you can just say "Search other users"

Reply:
DONE